# Action Creation/Navigation
string CREATE_ACTION = create-action
string SWITCH_TO_ACTION = switch-to-action
string NEXT_ACTION = next-action
string PREV_ACTION = previous-action
string UPDATE_ACTION_NAME = update-action-name
string DELETE_CURRENT_ACTION = delete-current-action
string DELETE_ACTION = delete-action
string COPY_ACTION = copy-action

# Primitive Creation Navigation
string SWITCH_PRIMITIVE_ORDER = switch-primitive-order
string DELETE_PRIMITIVE = delete-primitive
string COPY_PRIMITIVE = copy-primitive
string SELECT_PRIMITIVE = select-primitive
string DELETE_ALL_PRIMITIVES = delete-all-primitives
string DELETE_LAST_PRIMITIVE = delete-last-primitive

# Programming
string OPEN_HAND = open-hand
string CLOSE_HAND = close-hand

string RECORD_OBJECTS= record-objects
string SAVE_TARGET = save-target

string START_RECORDING_TRAJECTORY = start-recording-trajectory
string STOP_RECORDING_TRAJECTORY = stop-recording-trajectory

# Execution
string EXECUTE_ACTION = execute-action
string STOP_EXECUTION = stop-execution
string EXECUTE_PRIMITIVE = execute-primitive

# Message fields
string command
string param
uint16[] list_params
